tests/withIsMobile.test.js

```diff
@@ -32,39 +35,39 @@ describe('withResponsive()', () => {
     });
 
     test('should pass the values to the `isMobile` and `isCalculated` props correctly', () => {
-        window.resizeTo(1024, 768);
+        window.resizeTo(959, 768);
         const MockComponent = jest.fn(() => <h1>Mock test</h1>);
         const MockComponentWithIsMobile = withIsMobile(MockComponent);
 
         render(
-            <ResponsiveProvider>
+            <ResponsiveProvider {...mockProps}>
                 <MockComponentWithIsMobile />
             </ResponsiveProvider>
         );
 
         expect(MockComponent).toHaveBeenLastCalledWith(
             {
-                isMobile: false,
+                isMobile: true,
                 isCalculated: true,
             },
             {}
         );
     });
 
     test('should pass the values to the `isMobile` and `isCalculated` props correctly and the additional props correctly', () => {
```

Comment:
I would merge this test with the above since the only difference is that this one also tests additional props.

Then I would create one more to test the desktop version `isMobile` = `false` at `>1024`

Reply:
done!